test/test-webdriver-plus.ts

```diff
@@ -51,18 +68,55 @@ describe('webdriver-plus', () => {
       assert.deepEqual(elemsMapped, ['World!', 'OK']);
     });
 
+    it('should wait for match with driver.findWait()', async function() {
+      // Add component after 10ms.
+      addDomDelayed(10, 'waitForIt');
```

Comment:
I think it's better to avoid hanging promises, since in case of failure, you'd get an unhandled rejection, which is harder to debug. I.e. you could have addDomDelayed have a return in the last line, and use `Promise.all()` to wait for both addDomDelayed and driver.findWait.